crowbar_framework/app/controllers/backup_controller.rb

```diff
@@ -0,0 +1,162 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class BackupController < ApplicationController
+  skip_before_filter :enforce_installer
+
+  def index
+    @backups = Backup.all_images
+  end
+
+  #
+  # Backup
+  #
+  # Provides the restful api call for
+  # /utils/backup   POST   Trigger a backup
+  def backup
+    bu = Backup.create(params[:filename])
```

Comment:
> Why do you create a different interface? Do it like on restore with .new and create the backup on save.

`save` is basically an alias for `valid?` as you suggested me and as you did it in HAWK.
If you look at `Backup.create`, it has some logic like creating the archive and so on. The `BackupModel` (the Object) cannot create itself right now.